spec/lib/saml/dslogon_user_spec.rb

```diff
@@ -149,6 +149,10 @@
         )
       end
 
+      it 'has a user_subtype' do
+        expect(described_instance.user_subtype).to eq('ssn')
+      end
```

Comment:
i'm lost... why? this is always going to be SSN... we want the dslogon loa level i think

Reply:
also can we get a spec for every subtype? to be renamed "account_type" or "account_level" instead